packages/typography/src/beta/BlockquoteBeta.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export type BlockquotePropsBeta = {
+  /** Ekstra klassenavn */
+  className?: string;
+} & React.DetailedHTMLProps<
+  React.BlockquoteHTMLAttributes<HTMLElement>,
+  HTMLQuoteElement
+>;
+
+export const BlockquoteBeta = ({
+  className,
+  ref,
```

Comment:
Tror vi må bruke react sin forwardRef-wrapper her for at ref skal fungere ordentlig. Vi kan teste om det nåværende fungerer, men _my spider senses are tingling_.

Reply:
Vet ikke hvordan det fungerte før (og om ref er nødvendig for typografi), men mulig vi trenger å legge til det på alle typografikomponenter.

Docs om forwardRef: https://react.dev/reference/react/forwardRef